assets/icons/opendash.xpm

```diff
@@ -0,0 +1,368 @@
+/* XPM */
```

Comment:
this is cool! but what exactly is it for?

Would it help if i created a version of the logo to help this generate "more nicely"? Maybe an svg would help, and to remove the shadows?

Reply:
@rsjudka Did send a picture on Slack. It's a desktop icon for on the raspberry pi to start IA from there

Reply:
@rsjudka If you have a more nicely version i can convert it to a xpm file. this is the format for desktop icons.

Reply:
oh so this xpm file generates the desktop icon, not the png? If so, based on your pic I think it looks just fine

I was going off the ascii image in the file, which looked a bit "low res" haha